source/dem/pw_contact_force.cc

```diff
@@ -98,5 +98,130 @@ PWContactForce<dim>::update_contact_information(
   contact_info.tangential_relative_velocity = tangential_relative_velocity;
 }
 
+
+template <int dim>
+void
+PWContactForce<dim>::get_force_torque()
+{
+  for (const auto &it : ForceOnWall)
+    {
+      std::cout << "Boundary " << it.first << " :\n"
+                << "F = " << it.second << "\nM = " << TorqueOnWall[it.first]
+                << "\n\n";
+    }
+}
```

Comment:
It would be a good idea to have a getter, but this is a printing function. Please transform it into a getter.

Reply:
What is a getter?